piksi_tools/console/console.py

```diff
@@ -227,7 +227,10 @@ class SwiftConsole(HasTraits):
     view = View(
         VSplit(
             Tabbed(
-                Item('tracking_view', style='custom', label='Tracking'),
+                Tabbed(
+                    Item('tracking_view', style='custom', label='CN0 Graph', show_label=False),
```

Comment:
I'd call is SV Signals or just Signals

Reply:
you're thinking "Signals" and "Skyplot" for the labels?